src/components/search/SearchResult.jsx

```diff
@@ -0,0 +1,38 @@
+import './../../styles/Result.scss';
+
+import React from 'react';
+
+export const SearchResult = ({
+  user,
+  htmlUrl,
+  title,
+  bodyText,
+  userName,
+  repoName,
+  issueAge,
+  labels,
+}) => {
+  const mappedLabels = labels.map(label => {
+    return (
+      <span key={label.id} style={{ color: `#${label.color}` }} className="result-label">
+        {label.name}
+      </span>
+    );
+  });
+
+  return (
+    <div className="result">
+      <img src={user.avatar_url} width="50px" alt="avatar" class="result-avatar" />
```

Comment:
Thanks for your improvements! Looks great. 

Does `class` work here or should it be `className`?